packages/app/package.json

```diff
@@ -18,42 +18,44 @@
     "lint:js": "eslint .",
     "lint": "npm run lint:hbs && npm run lint:js",
     "start": "ember serve",
+    "pretest": " ember backstop-remote& ",
     "test": "export COVERAGE=true; ember test --test-port 0",
+    "posttest": "ember backstop-stop",
     "deploy-demo-app": "export BUILD_NAVI_DEMO=true; ember github-pages:commit --message 'update gh-pages' --destination ../../; echo 'gh-pages updated, now run:\ngit push origin gh-pages:gh-pages'"
   },
   "dependencies": {
     "compression": "^1.6.2",
-    "ember-auto-import": "^1.5.3",
     "express": "^4.14.0",
     "merge": "~1.2.0"
   },
   "devDependencies": {
     "@ember/jquery": "^0.6.1",
     "@ember/optional-features": "^1.1.0",
     "@glimmer/component": "^1.0.0",
-    "babel-eslint": "^10.0.3",
+    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^1.5.3",
-    "ember-basic-dropdown": "1.1.3",
+    "ember-auto-import": "^1.6.0",
+    "ember-backstop": "holderb5/ember-backstop#docker-support",
+    "ember-basic-dropdown": "^1.1.3",
     "ember-cli": "~3.15.2",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^7.17.2",
-    "ember-cli-code-coverage": "^1.0.0-beta.8",
+    "ember-cli-babel": "^7.21.0",
+    "ember-cli-code-coverage": "^1.0.0-beta.9",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-dependency-lint": "^1.1.3",
     "ember-cli-eslint": "^5.1.0",
-    "ember-cli-flash": "^1.8.1",
+    "ember-cli-flash": "^1.9.0",
```

Comment:
Are these version changes intentional?

Reply:
reverted the ones that were not intentional.